website/notifications/views.py

```diff
@@ -0,0 +1,51 @@
+from framework.auth.decorators import must_be_logged_in
+from model import Subscription
+from flask import request
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+from modularodm.storage.mongostorage import KeyExistsException
+from website import settings
+
+@must_be_logged_in
+def subscribe(auth):
```

Comment:
Since the name of the a view function determines the key for reverse lookup with `url_for` we may want to name this something more specific. Something `set_subscription` or `configure_subscription`? Look around the existing routes to find a name that is consistent.